infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsnext/KubernetesWorkspaceNextApplier.java

```diff
@@ -15,6 +15,7 @@
 import static java.util.Collections.singletonMap;
 import static java.util.stream.Collectors.toMap;
 import static org.eclipse.che.api.core.model.workspace.config.MachineConfig.MEMORY_LIMIT_ATTRIBUTE;
+import static org.eclipse.che.api.core.model.workspace.config.MachineConfig.MEMORY_REQUEST_ATTRIBUTE;
```

Comment:
Could you avoid changing this class? It is subject to change because we are experimenting with this stuff (it is not part of the regular user flow). So changing it will cause a lot of troubles in your PR

Reply:
should i rename the che.workspace.default_memory_mb property, or rely on alias to keep this code working ?